### PR TITLE
Fix parser import

### PR DESCRIPTION
**Pull request by vandewilly**
- Remove gographviz/parser import
- Move third-party libraries to vendor folder
- Fix graphviz and ovsdb imports

### CODE REVIEW

**Comment by davrodpin** [comment]
@vandewilly, PTAL

**Comment by vandewilly** [comment]
@davrodpin Is there any fix to be implemented in this review?

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 ## How to build and run
 
 ```
-$ cd $GOPATH
-$ go get github.com/awalterschulze/gographviz
-$ cd ./src/github.com/davrodpin/ovsdbviz # you have to clone or go get it
+$ cd $GOPATH/src
+$ git clone https://github.com/davrodpin/ovsdbviz.git
+$ cd ovsdbviz
```
Comment (davrodpin) [inline README.md]: The ovsdbviz source directory inside GOPATH should follow the whole github repo address:

```
$ cd $GOPATH/src
$ mkdir -p github.com/davrodpin && cd github.com/davrodpin
$ git clone https://github.com/davrodpin/ovsdbviz.git
$ cd ovsdbviz
```

File: main.go [hunk 2]
```diff
@@ -3,9 +3,9 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/davrodpin/ovsdbviz/graphviz"
-	"github.com/davrodpin/ovsdbviz/ovsdb"
 	"os"
+	"ovsdbviz/graphviz"
+	"ovsdbviz/ovsdb"
```
Comment (davrodpin) [inline main.go]: I think this change needs to be reverted. Both imports should include the whole github address.

**Review by davrodpin — CHANGES_REQUESTED** [review]

**Comment by vandewilly** [comment]
@davrodpin PTAL

**Review by davrodpin — APPROVED** [review]

**Pull request closed without merging**